CHANGELOG.md

```diff
@@ -41,12 +41,14 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Changed
 
-- The MSRV is now 1.56.1
-  ([#1792](https://github.com/nix-rust/nix/pull/1792))
+- The MSRV is now 1.58.1
+  ([#1876](https://github.com/nix-rust/nix/pull/1876))
```

Comment:
You can delete this part, and all of the other MSRV stuff, since we'll merge #1862 first.

Reply:
Dropped this commit